.github/workflows/action.yml

```diff
@@ -65,6 +65,68 @@ jobs:
       - name: Run unit tests
         run: yarn test
 
+  integration-tests:
```

Comment:
```suggestion
  integration-tests-modules:
```

Reply:
yeah I know, atm we only have integration tests in the modules though. How about "integration-tests-packages" then? 

just naming it "integration-tests" is a bit misleading imo

Reply:
yes sure, I can move it, I thought since we add api - plugins - repositories, the regular one would suffice 😂 i ll change it